src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IepLevelResourceTest.kt

```diff
@@ -96,51 +92,6 @@ class IepLevelResourceTest : SqsIntegrationTestBase() {
       )
   }
 
-  @Test
```

Comment:
What happened to this test and the other to test these two endpoints?

These endpoints are still there AFAICT
- `GET /iep/reviews/booking/{bookingId}`
- `POST /iep/reviews/bookings`

Reply:
Oh the tests are simply moved to `IepLevelResourceForNomisDataTest.kt` 👌